### ISSUE TITLE
Unnecessary relu applied to action probability logits

### ISSUE THREAD

**Issue by msokoloff1** [NVlabs/GA3C#17]
In the NetworkVP.py file on line 93, the activation function should be explicitly set to None. As it currently stands, the logits are being put through a relu non-linearity before sotfmax is applied.
